src/workerd/server/workerd-api.c++

```diff
@@ -446,7 +446,9 @@ void WorkerdApi::compileModules(
     if (hasPythonModules(confModules)) {
       KJ_REQUIRE(featureFlags.getPythonWorkers(),
           "The python_workers compatibility flag is required to use Python.");
-      // Inject pyodide bootstrap module.
+      // Inject Pyodide bundle
+      modules->addBuiltinBundle(PYODIDE_BUNDLE, kj::none);
```

Comment:
The `PYODIDE_BUNDLE` will get replaced with `pyodideBundleGlobal` here, right? Maybe add a todo.

Reply:
Yeah that's a todo, I guess @garrettgu10 is working on it =)